examples/eval-tests.dx

```diff
@@ -180,7 +180,7 @@ infty = 1./0.
 :p infty
 > Infinity
 
-nan = 0./0.
+--nan = 0./0.
 :p nan
 > NaN
```

Comment:
I removed the commented-out line, because I now introduce `nan` in the prelude.   But I think it makes sense to test how `nan` is printed out.  We might also want to make the capitalization of its printed representation match its name.